### DISCUSSION TITLE
RTP video stream + HDMI Display, how to configure on 3536 ground end?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#343] [Questions] [answered]
Can a RTP video stream be configured for other devices to access while outputting HDMI?

How to configure using the current release fw? Or need other scripts?

**Comment by svpcom** [comment, marked as answer]
Just connect them to the internal rtsp server on GS

**Reply by lida2003** [reply]
Oh, Thanks. I see the release note: [TSP server now enabled by default. So you can see video simultaneously on RPI (via HDMI) and on laptop via QGC](https://github.com/svpcom/wfb-ng/releases/tag/wfb-ng-23.01)
